src/layouts/sidebarLayout.tsx

```diff
@@ -13,23 +15,75 @@ const Container = styled.div`
   color: var(--indoor);
 `;
 
+const DesktopSidebar = styled(Sidebar)`
+  display: none;
+
+  @media only screen and (min-width: 700px) {
+    display: unset;
+  }
+`;
+
 const Main = styled.main`
+  display: none;
+
+  @media only screen and (min-width: 700px) {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 0 20px;
+    padding-left: calc(var(--sidebar-width) + 20px);
+  }
+`;
+
+const MobileView = styled.main`
   flex: 1;
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: flex-start;
-  padding: 0 20px;
-  padding-left: calc(var(--sidebar-width) + 20px);
+  justify-content: center;
+
+  background-color: var(--wineDark);
+  color: var(--white);
+
+  @media only screen and (min-width: 700px) {
+    display: none;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 16px;
 `;
 
 interface LayoutProps {
   children: React.ReactNode;
 }
 
-export default ({children}: LayoutProps) => (
-  <Container>
-    <Sidebar />
-    <Main>{children}</Main>
-  </Container>
-);
+export default ({children}: LayoutProps) => {
+  const history = useHistory();
+  const {signOut} = useAuth();
+
+  const handleSignOut = async () => {
+    await signOut();
+    history.push('/');
+  };
+
+  return (
+    <Container>
+      <DesktopSidebar />
+      <Main>{children}</Main>
+      <MobileView>
+        <h1>Screen Too Small</h1>
+        <p>Please sign in on a device with a larger screen.</p>
```

Comment:
This seems like an excessively blunt way to communicate this.
At least mentioning "mobile" would help.